tests/test_things.py

```diff
@@ -204,7 +206,7 @@ def test_database_version(self):
 
     def test_last(self):
         """Test last parameter"""
-        last_tasks = things.last("1d")
+        last_tasks = things.last("2d")
```

Comment:
😅

Reply:
Maybe change to 1d and assertEqual to 0? Otherwise, this test will keep changing.